### DISCUSSION TITLE
Authorizing a GitHub App returns 404 /app-manifests/{code}/conversions

### DISCUSSION THREAD

**Discussion by AceissBob** [community/community#69138] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Im following the [documentation](https://docs.github.com/en/free-pro-team@latest/rest/apps/apps?apiVersion=2022-11-28#create-a-github-app-from-a-manifest
) to Authorize a GitHub App. 

It states to POST the code to the url mentioned above. 
the url returns a 404. 

Is there something I'm missing? Are the docs inaccurate? I did see a similar post, but his solution was dealing with an enterprise server and adding /api/v3 (which I also tried)

**Comment by naveenOnarayanan** [comment]
@AceissBob did you end up getting this resolved? I'm hitting the same issue so curious if you had some workaround for this
